firmware/init/sensor/init_tps.cpp

```diff
@@ -156,19 +148,13 @@ printf("init m_redund.Register() %s\n", getSensorType(m_redund.type()));
 	RedundantSensor m_redund;
 };
 
-static FuncSensPair tps1p(TPS_TS_CONVERSION, SensorType::Tps1Primary);
-static FuncSensPair tps1s(TPS_TS_CONVERSION, SensorType::Tps1Secondary);
-static FuncSensPair tps2p(TPS_TS_CONVERSION, SensorType::Tps2Primary);
-static FuncSensPair tps2s(TPS_TS_CONVERSION, SensorType::Tps2Secondary);
-
-// Used in case of "normal", non-Ford ETB TPS
-static RedundantPair tps1(tps1p, tps1s, SensorType::Tps1);
-static RedundantPair tps2(tps2p, tps2s, SensorType::Tps2);
-
-// Used only in case of weird Ford-style ETB TPS
-static RedundantFordTps fordTps1(SensorType::Tps1, SensorType::Tps1Primary, SensorType::Tps1Secondary);
-static RedundantFordTps fordTps2(SensorType::Tps2, SensorType::Tps2Primary, SensorType::Tps2Secondary);
-static RedundantFordTps fordPps(SensorType::AcceleratorPedal, SensorType::AcceleratorPedalPrimary, SensorType::AcceleratorPedalSecondary);
+// Throttle sensors and redundancy
+static FuncSensPair throttle1Primary(TPS_TS_CONVERSION, SensorType::Tps1Primary);
```

Comment:
i like this rename into more human-readable names!

i should have merged this before SENT change has created a merge conflict. sounds like we need smaller less stuck commits

overall are we failing in something with all the TPS changes?